pkg/apis/kops/validation/validation.go

```diff
@@ -842,6 +846,18 @@ func validateKubeControllerManager(v *kops.KubeControllerManagerConfig, c *kops.
 	return allErrs
 }
 
+func validateKubeScheduler(v *kops.KubeSchedulerConfig, c *kops.Cluster, fldPath *field.Path, strict bool) field.ErrorList {
+	allErrs := field.ErrorList{}
+
+	// We aren't aiming to do comprehensive validation, but we can add some best-effort validation where it helps guide users.
+	// Users reported encountered this in #16388
+	if v.UsePolicyConfigMap != nil {
+		allErrs = append(allErrs, field.Forbidden(fldPath.Child("usePolicyConfigMap"), "usePolicyConfigMap is deprecated, use KubeSchedulerConfiguration"))
```

Comment:
Should we make this a warning in <1.23 and an error in >=1.23?  (Similar to my previous comment, I'm worried about "forcing" users to stay on an older version of kOps)

Reply:
We don't support K8s 1.23 anymore :)